go/src/helloworld/helloworld.go

```diff
@@ -3,5 +3,5 @@ package main
 import "fmt"
 
 func main() {
-    fmt.Println("hello world")
+    fmt.Println("hello world!")
```

Comment:
Oops. :)
On Jun 18, 2016 15:24, "Deniz Kennedy" <notifications@github.com> wrote:

> In go/src/helloworld/helloworld.go
> <https://github.com/naslundx/language-benchmarker/pull/24#discussion_r67600948>
> :
>
> > @@ -3,5 +3,5 @@ package main
> >  import "fmt"
> >
> >  func main() {
> > -    fmt.Println("hello world")
> > +    fmt.Println("hello world!")
>
> Hello instead of hello here.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/naslundx/language-benchmarker/pull/24/files/a5f3a83d2b273f0bc88f0b622a8652b80a05415b#r67600948>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACFvhFJGhww0cXs3N83gKKf40ZPnkJttks5qM_GUgaJpZM4I47Ew>
> .
>